spec/lib/zoom/token_store_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Zoom::TokenStore do
+  subject { described_class.build(config) }
+
+  let(:access_token) { 'xxx' }
+  let(:refresh_token) { 'xxx' }
+  let(:expires_in) { 3599 }
```

Comment:
более ожидаемое значение)
```suggestion
  let(:expires_in) { 3600 }
```